packages/contracts/src/contract-defs.ts

```diff
@@ -1,6 +1,11 @@
 import { ethers } from 'ethers'
-
-export const getContractDefinition = (name: string): any => {
+/** 
+ * Gets the contract's artifact
```

Comment:
It's a nit, but I think this should have a period after it for consistency. 

This is what was done in the example given in @smartcontracts review:
https://github.com/ethereum-optimism/optimism/blob/3525635a3b240a92434016d2129534892ad24124/packages/core-utils/src/optimism/alias.ts#L14-L19

Reply:
Got it!

Reply:
Thank you for pointing that out.

Reply:
There are merge conflicts.. Am I to resolve that?

Reply:
Yes please.

Reply:
To do that, I would: 
1. fix the conflicts in package.json
2. rerun `yarn`, which will overwrite the yarn.lock file with the correct contents.

Also don't forget to combine the commits per the [comment above](https://github.com/ethereum-optimism/optimism/pull/2303#pullrequestreview-906171765). :)

Reply:
Thanks! I have made the updates

Reply:
Checks are failing. What can I do?